docs/install.rst

```diff
@@ -3,7 +3,7 @@ Installation
 ************
 Requirements
 ~~~~~~~~~~~~
-Make sure you have installed Python `3.8` by typing
+Make sure you have installed Python `3.9` by typing
```

Comment:
The line below (not changed so cannot comment there)  "Consider using the python version management `pyenv`_." should be changed. pyenv should not be used in conjuction with conda. So maybe something like: 
If you do not (want to) use conda, consider using a python version management like pyenv.